app/templates/index.hbs

```diff
@@ -2,23 +2,14 @@
   {{latest-contribution}}
   <div class={{if (media "isDesktop") "layout-row" "layout-column"}}>
     <div class="page-index-content intro flex text-center">
-      <h1>{{this.model.name}}</h1>
+      <h1>Keine Open Source Advents Challenge 2020</h1>
       <p>
-        Stell dich der Herausforderung: Leiste jeden Tag im Advent (1.12. - 24.12.)
-        einen Beitrag zu einem Open Source Projekt und unterstütze damit
-        die Community.
+        Dieses Jahr verzichten wir auf die Durchführung einer Open Source Advents Challenge. Wir sind abei uns Gedanken über eine neues Format zu machen. 
+        Hast du eine Idee? Gerne nehmen wir deine Ideen und Vorschläge auf <a href="https://twitter.com/osschallengech">@osschallengech</a> entgegen.
       </p>
 
       <p>
-        Wichtig: Du musst kein Programmierer sein, um teilzunehmen! Auch
-        Textbeiträge oder eine Ortschaft auf <a href="http://www.osm.org/">OpenStreetMap</a> zu erfassen
-        ist bereits ein wertvoller Beitrag.
-      </p>
-
-      <p>
-        Keine Idee was du im Advent an der Open Source Challenge tun sollst? Oder hast du ein spannendes Projekt
-        und suchst Unterstützung? Wir haben ein <a href="https://wekan.puzzle.ch/b/aorgs9aHXn6Prhqnn/opensource-challenge-2018-ideen">Wekan Board</a>
-        erstellt. Für Schreibrechte bitte pn mit deiner Mailadresse an <a href="https://twitter.com/osschallengech">@osschallengech</a>.
+        Die vergangen Challenges findest du im {{#link-to "arcive"}}«Archiv»{{/link-to}}.
```

Comment:
Typo, `arcive` -> `archive`

Reply:
thanks

Reply:
Not really fixed yet, this creates a broken link 🙃